snippets/shiny.json

```
@@ -20,7 +20,23 @@
       "app = App(app_ui, server)",
       ""
     ],
-    "description": "Basic Shiny app skeleton"
+    "description": "Basic Shiny Core app skeleton"
+  },
+  "Shiny Express application": {
+    "scope": "python",
+    "prefix": "shinyexpress",
+    "body": [
+      "from shiny.express import input, render, ui",
+      "",
+      "ui.input_slider(\"n\", \"N\", 0, 100, 20)",
+      "",
+      "",
+      "@render.text",
+      "def txt():",
+      "\t\treturn f\"n*2 is {input.n() * 2}\"",
```

Is this too many tabs?

---

Yup, I must have been thinking in R for a second there.

```suggestion
      "\treturn f\"n*2 is {input.n() * 2}\"",
```